src/UI/Actions.hs

```diff
@@ -301,7 +301,7 @@ reloadMails = Action "reload list of mails" applySearch
 --
 applySearch :: AppState -> T.EventM Name AppState
 applySearch s =
-   runExceptT (Notmuch.getMessages searchterms (view (asConfig . confNotmuch) s))
+   runExceptT (Notmuch.getTopMessages searchterms (view (asConfig . confNotmuch) s))
```

Comment:
I'm unsure about whether we really want this to be the default or not.  I can forsee a need to make it configurable in the future.  But I'm ok with the change for now.  I'm not even using purebred in anger yet... you are (probably) the only person in the world who is right now, so I'll defer to your judgement :)

Reply:
Yeah I'm actually not sure about this either. Most clients follow the approach of showing the conversation. That is: thread -> unread mails expanded with folded quoting. I grew up with mutt showing individual mails. Thinking about it however I can't think of a case where I would want to use a thread list with only one mail expanded (mutt style). Do you have a good one?

Reply:
Nope.  Go with your gut, I'll go along with it :)